pre-piksi-multi-launch-updates.md

```diff
@@ -0,0 +1,121 @@
+### SBP changes before shipping Piksi Multi
+
+There are a few changes we would like to make in the SBP protocol. These changes
+will be much more painful to make after we ship and customers are relying on
+the protocol to be largely static, so we'd like to come to a consensus on
+these and implement them before shipping Piksi Multi.
+
+These changes attempt to accomplish a few things:
+- Make messages more bandwidth efficient
+- Make user side logic of using messages more obvious
+- Remove state from the protocol, where possible
+
+### 1. Replace obs forwarding with general purpose forwarding message
+
+Current there is a convention in the protocol that if the sender_id of a
```

Comment:
The idea was to encapsulate a SBP message in the payload of an SBP message - no preamble, no crc. This should work ok with the parser, though how to express that in the yaml is not clear - it would need to be a special rule or something.

And I don't think this message would be forwarded on, but not sure about that. If it were to be forwarded on, would need to TTL for sure. If not, would be another exceptional rule to cover this case.

Maybe all these exceptions make this not such a sound idea.